java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture5/tasks/_1/Number1.java

```diff
@@ -10,75 +10,29 @@ The method needs to return the value of the Nth (1-based) smallest element in th
 Output: -1
 */
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
 
 public class Number1 {
-    public static void main(String[] args) {
-
-        ArrayList<Integer> arrayList = new ArrayList<>();
-        arrayList.add(1);
-        arrayList.add(5);
-        arrayList.add(2);
-        arrayList.add(3);
-        arrayList.add(7);
-
-        int N = 4;
+    ArrayList<Integer> arrayList = new ArrayList<>();
```

Comment:
If you define a list as an instance variable, each call to the method will add elements to the same instance of the list. It will be better to define it inside of the method.